node.go

```diff
@@ -1,11 +1,80 @@
 package pgs
 
+import (
+	"strings"
+
+	"github.com/golang/protobuf/protoc-gen-go/descriptor"
+)
+
 // Node represents any member of the proto descriptor AST. Typically, the
 // highest level Node is the Package.
 type Node interface {
 	accept(Visitor) error
 }
 
+// NodeMeta contains metadata for a node.
+//
+// See the documentation for .google.protobuf.SourceCodeInfo for more details.
+//
+// https://github.com/google/protobuf/blob/master/src/google/protobuf/descriptor.proto
+type NodeMeta struct {
+	*descriptor.SourceCodeInfo_Location
+}
+
+// Line returns the starting line for the node, or 0 if unknown.
```

Comment:
Line/Column values in the Span are zero-indexed, so we should use a different value for unknown (-1?).

Reply:
We should also add comments to these methods wrt zero-indexing.

Reply:
Shoot I missed that in the documentation - I'd prefer actually to make these not zero-indexed - for the intended usage for linters and formatters, and just for any general use for line and column numbers, you want the actual line number. Maybe instead I'll return `value + 1`, with `0` still as the default?

`NodeMeta` has the `SourceCodeInfo_Location` on it exposed, so users who want the native underlying functionality can still access it.

Reply:
I'm cool with that